backend/climateconnect_api/utility/email_setup.py

```diff
@@ -272,3 +278,117 @@ def remove_contact_from_list(contact_id, list_id):
+            "location": project.loc.name if project.loc else '',
+            "creator": '',
+            "tags": '',
+        }
+        entities.append(project_template)
+    for organization_id in organization_ids:
+        organization = Organization.objects.select_related('location').get(id=organization_id)
+        orga_template = {
+            "type": organization_translation.get(language_code, "en"),
+            "name": organization.name,
+            "imageUrl": (settings.ALLOWED_HOSTS[6] + organization.thumbnail_image.url) if organization.thumbnail_image else '',
+            "url": (settings.FRONTEND_URL + "/organizations/" + organization.url_slug) if organization.url_slug else main_page.get(language_code, "en"),
+            "location": organization.location.name if organization.location else '',
+            "creator": '',
+            "tags": '',
+        }
+        entities.append(orga_template)
+    for idea_id in idea_ids:
+        idea = Idea.objects.select_related('location', 'user').get(id=idea_id)
+        idea_template = {
+            "type": idea_translation.get(language_code, "en"),
+            "name": idea.name,
+            "imageUrl": (settings.ALLOWED_HOSTS[6] + idea.thumbnail_image.url) if idea.thumbnail_image else '',
+            # How does the idea url generation work
+            "url": main_page.get(language_code, "en"), #(settings.FRONTEND_URL + "/idea/" + idea.url_slug) if idea.url_slug else main_page.get(language_code, "en"),
+            "location": idea.location.name if idea.location else '',
+            "creator": '',
+            "tags": '',
+        }
+        entities.append(idea_template)
+    return entities
+
+
+def send_weekly_personalized_recommendations_email(user: User, project_ids: List = [], organization_ids: List = [], idea_ids: List = []):
+
+    template_key = "WEEKLY_RECOMMENDATIONS_EMAIL"
+
+    lang_code = get_user_lang_code(user)
+    
+    template_id = get_template_id(
+        template_key=template_key,
+        lang_code=lang_code
+    )
+
+    entities = create_variables_for_weekly_recommendations(lang_code, project_ids, organization_ids, idea_ids) 
+
+    variables =  {
+        "FirstName": user.first_name,
+        "Entities": entities,
+    }
+
+    data = {
+        'Messages': [
+            {
+                "From": {
+                    "Email": settings.CLIMATE_CONNECT_SUPPORT_EMAIL,
+                    "Name": "Climate Connect"
+                },
+                "To": [
+                    {
+                        "Email": user.email,
+                        "Name": user.first_name + " " + user.last_name
+                    }
+                ],
+                "TemplateID": int(template_id),
+                "TemplateLanguage": True,
+                "Variables": variables,
+                "TemplateErrorReporting": {
+                    "Email": "philipp.ernstberger@climateconnect.earth",
```

Comment:
Maybe we could use an env variable for this so we can use different emails in prod and dev environments without having to change the code.